docs/shared/reference/environment-variables.md

```diff
@@ -25,8 +25,7 @@ The following environment variables are ones that you can set to change the beha
 | NX_GENERATE_QUIET                | boolean | If set to `true`, will prevent Nx logging file operations during generate                                                                                                                                                              |
 | NX_PREFER_TS_NODE                | boolean | If set to `true`, Nx will use ts-node for local execution of plugins even if `@swc-node/register` is installed.                                                                                                                        |
 | NX_IGNORE_CYCLES                 | boolean | If set to `true`, Nx will ignore errors created by a task graph circular dependency. Can be overriden on the command line with `--nxIgnoreCycles`                                                                                      |
-
-Nx will set the following environment variables so they can be accessible within the process even outside of executors and generators
```

Comment:
This shouldn't be dropped. Can you add it back?

Reply:
Oops, good catch. Done.